### ISSUE TITLE
Adding new gateway

### ISSUE THREAD

**Issue by actnow** [eileenmcnaughton/nz.co.fuzion.omnipaymultiprocessor#63]
I get this error: 
Sorry, due to an error, we are unable to fulfill your request at the moment. You may want to contact your administrator or service provider with more details about what action you were performing when this occurred.
Class '\Omnipay\mygateway\Gateway' not found

**Comment by eileenmcnaughton** [comment]
I probably need a bit more detail about what you did to add mygateway....

Did you run composer?

**Comment by actnow** [comment]
Yes added the below lines, and ran "composer update" based on https://stackoverflow.com/questions/26179177/omnipay-add-new-gateway add "autoload" too
It installs without error

"razorpay/omnipay-razorpay": "dev-master"
    },
        "autoload": {

        "psr-0": { 
            "": "lib/", 
            "Omnipay\\Razorpay\\" : "lib/omnipay/src"
        }
    },

**Comment by eileenmcnaughton** [comment]
so above - should that say 

```
Class '\Omnipay\Razorpay\Gateway' not found
```

I can't see where 'mygateway' would come from

**Comment by actnow** [comment]
Sorry the error is Class '\Omnipay\Razorpay\Gateway' not found
I tried to find solution..

**Comment by eileenmcnaughton** [comment]
Hmm - I can take a look tomorrow if you push your code up as a PR so I can download it

**Comment by actnow** [comment]
Thanks will do
